app/assets/javascripts/proceeding_types.es6

```diff
@@ -1,65 +1,83 @@
-var submitForm = proceedingItem => {
-  $(proceedingItem).find('form').submit();
-}
-
-$.getJSON("/v1/proceeding_types", function (proceedings_data) {
+if ($(document).text().includes('Search for legal proceedings')){
```

Comment:
I suggest using the ID of the input rather than text:
`if ($("#proceeding-search-input").length) {`

Reply:
Thanks. Fixed.